lighthouse-core/report/html/report-styles.css

```diff
@@ -679,6 +679,19 @@
   vertical-align: middle;
 }
 
+.lh-audit-group--budgets .lh-table tbody tr td:nth-child(4),
+.lh-audit-group--budgets .lh-table tbody tr td:nth-child(5){
```

Comment:
might as well do this in the next PR where you add support for the 'over budget' column?

already thinking you could use a `.lh-table-column--overbudge` class instead of nth-child.

Reply:
This PR "has" the over budget column (see below for what it looks like when all the branches are together) in the sense that it renders it, but @brendankenny wanted me to split the renderer code from the audit code for PR purposes.

Agreed that this CSS is kinda gross. However, I did it that way because it allows the budgets table to re-use Audit.makeTableDetails() so there's really no custom code - just the code in this branch which all reuses existing methods. If you think this is worth changing, I'd be happy to do that.

With a budget:
// Only lists resources that have budgets associated with them.
<img width="912" alt="Screen Shot 2019-04-23 at 2 35 19 AM" src="https://user-images.githubusercontent.com/6076184/56926056-cdb3cb00-6a9e-11e9-93b6-bff3edf1275e.png">

Without a budget:
// Lists all resources
<img width="938" alt="Screen Shot 2019-04-23 at 2 26 29 AM" src="https://user-images.githubusercontent.com/6076184/56926027-bb399180-6a9e-11e9-9434-5f8fcead7bec.png">

Reply:
whats the easiest way for me to generate that first screenshot of yours?

Reply:
The easiest way would be to use the khempenius/lightwallet_pt2 branch.

That was the branch that I used to split this out into separate PRs. It doesn't have some of the minor code cleanup changes that have occurred on the PR branches, but is functionally the same and has all the upstream dependencies necessary to generate this.